design/templates/solution_design.md

```diff
@@ -67,5 +70,41 @@
 ## Open questions
 [//]: # (Add any question that is still open. It makes it easier for the reader to have the open questions accumulated here istead of them being acattered along the doc)
 
-## Implementation plan
+## Delivery plan
 [//]: # (Break the solution into tasks)
+
+### Total
+[//]: # (Write the total estimation)
+
+### Parallelism
+[//]: # (Suggest how many team members can work on the solution in parallel)
+
+### Ramp up
+[//]: # (Estimate the tasks, which related of leaning new area [technology, language, feature, etc.])
+
+### Infrastructure and preparations
+[//]: # (Estimate the tasks, which related of implementing any needed infrastructure [jenkins file, scripts, Docker build, etc.])
```

Comment:
`Estimate the tasks that are related to implementing any needed infrastructure [jenkins file, scripts, Docker build, etc.]`

Reply:
does this include requirements from infra? For example, a new GCE node. if so, it is not clear to me from the description.

Reply:
No, because we are not calculating the effort of other teams, we just need to verify they will finish their tasks on time